### PR TITLE
chore: E2E refactoring.

### PR DESCRIPTION
**Pull request by jparsai**
## Description:
This PR is to refactor Environment E2E tests and remove TODO comment that is no longer required now.

## JIRA Story:
https://issues.redhat.com/browse/GITOPSRVCE-540
https://issues.redhat.com/browse/GITOPSRVCE-85

### CODE REVIEW

**Review by keithchong — APPROVED** [review]
@jparsai , LGTM.  From what I can tell, it is strictly refactoring.

File: tests-e2e/core/managed_environment_test.go [hunk 1]
```diff
@@ -1,1189 +0,0 @@
-package core
```
Comment (jgwest) [inline tests-e2e/core/managed_environment_test.go:1]: The goal of GITOPSRVCE-540 was actually only to move the `Environment E2E tests` part of this file, into `environment_test.go`. The remaining tests can thus stay within `managed_environment_test.go`.

The overall goal is:
- tests primarily for ManagedEnvironments go in `managed_environment_test.go`
- tests primarily for (AppStudio) Environment go in `environment_test.go`

**Review by jgwest — CHANGES_REQUESTED** [review]

**Comment by jgwest** [comment]
Rebased to latest

**Review by jgwest — APPROVED** [review]
LGTM, thanks @jparsai, and thanks @keithchong for reviewing!

**Pull request merged**